python/postprocessing/modules/jme/jetmetUncertainties.py

```diff
@@ -76,21 +72,13 @@ def __init__(self, era, globalTag, jesUncertainties = [ "Total" ], jetType = "AK
         self.jesArchive.extractall(self.jesInputFilePath)
         
         if len(jesUncertainties) == 1 and jesUncertainties[0] == "Total":
-            if self.era == "2016":
-                self.jesUncertaintyInputFileName = "Summer16_07Aug2017_V11_MC_Uncertainty_" + jetType + ".txt"
-            elif self.era == "2017":
-                self.jesUncertaintyInputFileName = "Fall17_17Nov2017_V32_MC_Uncertainty_" + jetType + ".txt"
-            elif self.era == "2018":
-                self.jesUncertaintyInputFileName = "Autumn18_V8_MC_Uncertainty_" + jetType + ".txt"
+            if self.era in ["2016", "2017", "2018"]:
+                self.jesUncertaintyInputFileName = self.jesUncertaintyInputFileName = globalTag + "_Uncertainty_" + jetType + ".txt"
```

Comment:
typo?

Reply:
@fgolf yes it was a bug. I fixed it in a new commit. I will wait for the other PR related the jets to be merged. After that I will rebase and make this PR again.